The STL headers in gcc-4.3 have been cleaned up, so that the headers don't drag in
unnecessary dependencies which aren't requested by the standard.
This fixes build for gcc-4.3

Lubomir Kundrak <lkundrak@redhat.com>

diff -urp ptouch-driver-1.2.orig/rastertoptch.c ptouch-driver-1.2/rastertoptch.c
--- ptouch-driver-1.2.orig/rastertoptch.c	2006-09-08 16:39:27.000000000 +0200
+++ ptouch-driver-1.2/rastertoptch.c	2008-02-13 16:37:13.000000000 +0100
@@ -311,6 +311,7 @@
 #include <math.h>
 #include <cups/raster.h>
 #include <cups/cups.h>
+#include <limits.h>
 
 #if STDC_HEADERS
 # include <string.h>
